src/service.spec.ts

```diff
@@ -198,39 +192,19 @@ describe('#handleConnectionClose', () => {
 
   it('reconnects and rebinds handlers', async () => {
     serviceConnection.unsubscribe = jest.fn().mockResolvedValue({});
-    const connection = { bindQueue: jest.fn() };
+    const connection = { queueBind: jest.fn() };
     serviceConnection.connect = jest.fn().mockResolvedValue(connection);
     serviceConnection.initQueue = jest.fn();
 
     const handlerMock = async () => undefined;
     serviceConnection.setActionHandler('handler1', handlerMock);
 
-    await serviceConnection.handleConnectionClose(message as unknown as Message);
+    await serviceConnection.handleConnectionClose({} as unknown as AMQPError);
 
     expect(serviceConnection.unsubscribe).toBeCalled();
     expect(serviceConnection.connect).toBeCalled();
     expect(serviceConnection.initQueue).toBeCalled();
-    expect(connection.bindQueue).toBeCalledWith('dispatcher', 'dispatcher', '*.handler1');
-  });
-});
-
-describe('#connectionEventHandler', () => {
-  const message = { content: 'message' };
-
-  it('calls connection error handler on error event', () => {
```

Comment:
Does connectionEventHandler not call connection error handler on error event for now?

Reply:
amqplib supported multiple connection and channel events which from actually handled only `close` event. Everthing else just logged. amqp-client.js supports only `close` event and I decided that dispatcher method `connectionEventHandler` is redundant. `handlerConnectionClose` is directly called in `onerror` callback,